java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifierTest.java

```diff
@@ -331,11 +337,11 @@ public void verify_flow_locations() {
       .issueWithFlow(20)
         .flow(17, "msg", 19, null)
         .add();
-    try {
-      JavaCheckVerifier.verify("src/test/files/JavaCheckVerifierFlows.java", fakeVisitor);
```

Comment:
There is a missing assert to make this fail when no issue is detected, that's why I reworked it to the new AssertJ style